src/javascript/audio.js

```diff
@@ -74,6 +74,7 @@ export default class Audio {
     let bgmUrlArr = [
       assetsBaseUrl + "aurora.mp3",
       assetsBaseUrl + "beyond.mp3",
+      // assetsBaseUrl + "machinery.mp3",
```

Comment:
Is this needed? If not, simply remove it?

```suggestion
```

Reply:
oof.. will need that very soon... So I'm gonna just leave this for now